services/web/client/source/class/osparc/component/widget/NodeTreeItem.js

```diff
@@ -146,8 +146,10 @@ qx.Class.define("osparc.component.widget.NodeTreeItem", {
         case "node-id": {
           control = new qx.ui.basic.Label().set({
             maxWidth: 70,
-            alignY: "middle"
+            alignY: "middle",
+            cursor: "copy"
           });
+          control.addListener("tap", () => osparc.utils.Utils.copyTextToClipboard(this.getNodeId()));
```

Comment:
OK, I see now. So this is "tapping" the text of the UUID, not a button like in the info pannel, right